### PR TITLE
Added Turtle games

### PR DESCRIPTION
**Pull request by sree-hari-s**
#14 I have added a few games I have created 
Snake Game
Pong Game
Turtle Racing Game
Hirst Painting Generator

### CODE REVIEW

File: turtle-race.py
Comment (eskayML) [inline turtle-race.py:1]: This one is quite enjoyable 💯🔥

File: hirst-painting.py
Comment (eskayML) [inline hirst-painting.py:1]: I think your line 26 might be causing a problem here , your turtle keeps drawing an invisible pen

**Comment by eskayML** [comment]
Love your commits so far .. I need you to help me change one more thing 

via convention we prefer  single file contributions

Is it possible to make your pong and snake game , to be single `.py` files each.

This is done so that other people exploring this repo can view & run your code easily

**Comment by sree-hari-s** [comment]
yes it is possible just made that way to understand the code better wanted to ask you regarding this. Will update soon

**Comment by sree-hari-s** [comment]
I have updated both the code to a single file

**Comment by eskayML** [comment]
Make another commit for your hirst-painting , it looks like you used `t.penup()` so nothing is being drawn

**Review by eskayML — COMMENTED** [review]
LGTM🔥🔥

File: hirst-painting.py
Comment (sree-hari-s) [reply hirst-painting.py:1]: It is meant that way so that you can see the cursor moving but not the trace of the path

Comment (eskayML) [reply hirst-painting.py:1]: Alright no problem then, 
All good 

Please Share the repo with new contributors as part of Hacktoberfest 🔥

Comment (sree-hari-s) [reply hirst-painting.py:1]: Sure will do

**Pull request merged**